--- libnl-3.2.21/tests/test-complex-HTB-with-hash-filters.c.org	2015-05-27 23:39:30.440240674 -0500
+++ libnl-3.2.21/tests/test-complex-HTB-with-hash-filters.c	2015-05-27 23:52:25.281424945 -0500
@@ -534,7 +534,7 @@ int main() {
     }
     
     /* lookup interface index of eth0 */
-    if (!(link = rtnl_link_get_by_name(link_cache, "imq0"))) {
+    if (!(link = rtnl_link_get_by_name(link_cache, "eth0"))) {
         /* error */
         printf("Interface not found\n");
         nl_socket_free(sock);

--- libnl-3.2.21/tests/test-nf-cache-mngr.c.org	2015-05-27 23:55:13.940553623 -0500
+++ libnl-3.2.21/tests/test-nf-cache-mngr.c	2015-05-27 23:56:56.785242090 -0500
@@ -27,6 +27,7 @@ int main(int argc, char *argv[])
 	struct nl_sock *sock;
 	struct nl_cache *ct;
 	int err;
+	int i;
 
 	sock = nl_cli_alloc_socket();
 
@@ -42,7 +43,7 @@ int main(int argc, char *argv[])
 		return -1;
 	}
 
-	for (;;) {
+	for (i=0;i<=5;i++) {
 		int err = nl_cache_mngr_poll(mngr, 5000);
 		if (err < 0) {
 			nl_perror(err, "nl_cache_mngr_poll()");

