diff -ur ppc64-diag-2.6.1_orig/lpd/test/lpd_ela_test.sh ppc64-diag-2.6.1/lpd/test/lpd_ela_test.sh
--- ppc64-diag-2.6.1_orig/lpd/test/lpd_ela_test.sh	2014-07-18 13:33:08.351629000 +0200
+++ ppc64-diag-2.6.1/lpd/test/lpd_ela_test.sh	2014-07-18 13:33:15.323767000 +0200
@@ -22,7 +22,7 @@
 ERROR_LOG="$LOG_DIR/$SCRIPT_NAME.err"
 
 TMP_DIR="/var/tmp/ras"
-mkdir -p $TMP_DIR
+mkdir $TMP_DIR || exit 1
 MESSAGE_FILE="$TMP_DIR/messages"
 TMP_FILE="$TMP_DIR/$SCRIPT_NAME.tmp"
 
diff -ur ppc64-diag-2.6.1_orig/scripts/ppc64_diag_mkrsrc ppc64-diag-2.6.1/scripts/ppc64_diag_mkrsrc
--- ppc64-diag-2.6.1_orig/scripts/ppc64_diag_mkrsrc	2014-07-18 13:33:08.189631000 +0200
+++ ppc64-diag-2.6.1/scripts/ppc64_diag_mkrsrc	2014-07-18 13:33:15.384788000 +0200
@@ -123,7 +123,7 @@
 }
 
 # Retrieve Extended Error Data (EED)
-mkdir "/tmp/diagSEsnap", 0775;
+mkdir "/tmp/diagSEsnap", 0700 || die "mkdir failed: $!";
 $general_eed_file = "/tmp/diagSEsnap/snapH.tar.gz";
 system("/usr/sbin/snap -o $general_eed_file 2>/dev/null 1>&2");
 $cstop_eed_file = "";
