diff -up ppc64-diag-2.6.7/ela/catalogs.h.msg_loc ppc64-diag-2.6.7/ela/catalogs.h
--- ppc64-diag-2.6.7/ela/catalogs.h.msg_loc	2014-08-16 07:35:55.000000000 +0200
+++ ppc64-diag-2.6.7/ela/catalogs.h	2014-08-25 11:00:51.209997360 +0200
@@ -21,7 +21,7 @@ using namespace std;
 #include <stdio.h>
 #include <regex.h>
 
-#define ELA_CATALOG_DIR "/etc/ppc64-diag/message_catalog"
+#define ELA_CATALOG_DIR "/usr/share/ppc64-diag/message_catalog"
 
 class Parser {
 protected:
diff -up ppc64-diag-2.6.7/ela/explain_syslog.cpp.msg_loc ppc64-diag-2.6.7/ela/explain_syslog.cpp
--- ppc64-diag-2.6.7/ela/explain_syslog.cpp.msg_loc	2014-08-16 07:35:55.000000000 +0200
+++ ppc64-diag-2.6.7/ela/explain_syslog.cpp	2014-08-25 11:00:51.209997360 +0200
@@ -97,7 +97,7 @@ print_help(void)
 	printf(
 "-b begin_time\tIgnore messages with timestamps prior to begin_time.\n"
 "-C catalog_dir\tUse message catalog in catalog_dir.  Defaults to\n"
-"\t\t\t/etc/ppc64-diag/message_catalog.\n"
+"\t\t\t/usr/share/ppc64-diag/message_catalog.\n"
 "-d\t\tPrint debugging output on stderr.\n"
 "-e end_time\tStop upon reading message with timestamp after end_time.\n"
 "-h\t\tPrint this help text and exit.\n"
diff -up ppc64-diag-2.6.7/ela/Makefile.msg_loc ppc64-diag-2.6.7/ela/Makefile
--- ppc64-diag-2.6.7/ela/Makefile.msg_loc	2014-08-25 11:00:51.209997360 +0200
+++ ppc64-diag-2.6.7/ela/Makefile	2014-08-25 11:06:55.054005936 +0200
@@ -67,8 +67,8 @@ catalogs.o: catalogs.h
 
 install: all
 	@$(call install_sbin,$(CMDS),$(DESTDIR))        
-	@$(call install_files,$(CATALOG),744,$(DESTDIR)/etc/ppc64-diag/message_catalog)
-	@$(call install_files,$(CATALOG_REGEX),744,$(DESTDIR)/etc/ppc64-diag/message_catalog/with_regex)
+	@$(call install_files,$(CATALOG),744,$(DESTDIR)/usr/share/ppc64-diag/message_catalog)
+	@$(call install_files,$(CATALOG_REGEX),744,$(DESTDIR)/usr/share/ppc64-diag/message_catalog/with_regex)
 	$(INSTALL) -d -m 755 $(DESTDIR)/var/log/ppc64-diag
 	@$(foreach d,$(SUBDIRS), $(MAKE) -C $d install;)
 
@@ -78,6 +78,6 @@ clean:
 
 uninstall:
 	@$(call uninstall_sbin,$(CMDS),$(DESTDIR))        
-	@$(call uninstall_files,$(CATALOG),$(DESTDIR)/etc/ppc64-diag/)
-	@$(call uninstall_files,$(CATALOG_REGEX),$(DESTDIR)/etc/ppc64-diag/)
+	@$(call uninstall_files,$(CATALOG),$(DESTDIR)/usr/share/ppc64-diag/)
+	@$(call uninstall_files,$(CATALOG_REGEX),$(DESTDIR)/usr/share/ppc64-diag/)
 	@$(foreach d,$(SUBDIRS), $(MAKE) -C $d uninstall;)
diff -up ppc64-diag-2.6.7/ela/syslog_to_svclog.cpp.msg_loc ppc64-diag-2.6.7/ela/syslog_to_svclog.cpp
--- ppc64-diag-2.6.7/ela/syslog_to_svclog.cpp.msg_loc	2014-08-16 07:35:55.000000000 +0200
+++ ppc64-diag-2.6.7/ela/syslog_to_svclog.cpp	2014-08-25 11:00:51.209997360 +0200
@@ -657,7 +657,7 @@ print_help(void)
 	printf(
 "-b begin_time\tIgnore messages with timestamps prior to begin_time.\n"
 "-C catalog_dir\tUse message catalog in catalog_dir.  Defaults to\n"
-"\t\t\t/etc/ppc64-diag/message_catalog.\n"
+"\t\t\t/usr/share/ppc64-diag/message_catalog.\n"
 "-d\t\tPrint debugging output on stderr.\n"
 "-e end_time\tStop upon reading message with timestamp after end_time.\n"
 "-F\t\tDon't stop at EOF; process newly logged messages as they occur.\n"
