--- crash-7.0.7/cmdline.c.orig
+++ crash-7.0.7/cmdline.c
@@ -44,7 +44,8 @@ static int verify_args_input_file(char *
 #define READLINE_LIBRARY
 
 #include <readline.h>
-#include <rldefs.h>
+#define vi_mode 0
+#define emacs_mode 1
 #include <history.h>
 
 static void readline_init(void);
--- crash-7.0.7/Makefile.orig
+++ crash-7.0.7/Makefile
@@ -205,7 +205,7 @@ TAR_FILES=${SOURCE_FILES} Makefile ${GPL
 	${EXTENSION_SOURCE_FILES} ${MEMORY_DRIVER_FILES}
 CSCOPE_FILES=${SOURCE_FILES}
 
-READLINE_DIRECTORY=./${GDB}/readline
+READLINE_DIRECTORY=/usr/include/readline
 BFD_DIRECTORY=./${GDB}/bfd
 GDB_INCLUDE_DIRECTORY=./${GDB}/include
 
@@ -233,7 +233,7 @@ gdb_merge: force
 	@rm -f ${PROGRAM}
 	@if [ ! -f ${GDB}/config.status ]; then \
 	  (cd ${GDB}; ./configure ${GDB_CONF_FLAGS} --with-separate-debug-dir=/usr/lib/debug \
-	    --with-bugurl="" --with-expat=no --with-python=no --disable-sim; \
+	    --with-bugurl="" --with-expat=no --with-python=no --disable-sim --with-system-readline; \
 	  make --no-print-directory CRASH_TARGET=${TARGET}; echo ${TARGET} > crash.target) \
 	else make --no-print-directory rebuild; fi
 	@if [ ! -f ${PROGRAM} ]; then \
