From 3bc5bbbee3fdedcd26047c5aaaa8c6b3d68503b9 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Thu, 10 Sep 2015 16:38:02 +0200
Subject: [PATCH] change 4001 to 2379 in help and README

---
 README.md | 2 +-
 main.go   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/README.md b/README.md
index 65b0c1f..d27d954 100644
--- a/README.md
+++ b/README.md
@@ -178,7 +178,7 @@ $ flanneld --remote=10.0.0.3:8888 --networks=blue,green
 
 ```
 --public-ip="": IP accessible by other nodes for inter-host communication. Defaults to the IP of the interface being used for communication.
---etcd-endpoints=http://127.0.0.1:4001: a comma-delimited list of etcd endpoints.
+--etcd-endpoints=http://127.0.0.1:2379: a comma-delimited list of etcd endpoints.
 --etcd-prefix=/coreos.com/network: etcd prefix.
 --etcd-keyfile="": SSL key file used to secure etcd communication.
 --etcd-certfile="": SSL certification file used to secure etcd communication.
diff --git a/main.go b/main.go
index 9e10455..88298bc 100644
--- a/main.go
+++ b/main.go
@@ -62,7 +62,7 @@ var opts CmdLineOpts
 
 func init() {
 	flag.StringVar(&opts.publicIP, "public-ip", "", "IP accessible by other nodes for inter-host communication")
-	flag.StringVar(&opts.etcdEndpoints, "etcd-endpoints", "http://127.0.0.1:4001,http://127.0.0.1:2379", "a comma-delimited list of etcd endpoints")
+	flag.StringVar(&opts.etcdEndpoints, "etcd-endpoints", "http://127.0.0.1:2379", "a comma-delimited list of etcd endpoints")
 	flag.StringVar(&opts.etcdPrefix, "etcd-prefix", "/coreos.com/network", "etcd prefix")
 	flag.StringVar(&opts.etcdKeyfile, "etcd-keyfile", "", "SSL key file used to secure etcd communication")
 	flag.StringVar(&opts.etcdCertfile, "etcd-certfile", "", "SSL certification file used to secure etcd communication")
-- 
1.9.3

