From c8df9ca6c4fc7d9dd9578278e6537b865a3532e4 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Mon, 29 Feb 2016 14:53:50 +0100
Subject: [PATCH] disable v1beta3

---
 pkg/api/install/install.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkg/api/install/install.go b/pkg/api/install/install.go
index eb15b54..b239dc5 100644
--- a/pkg/api/install/install.go
+++ b/pkg/api/install/install.go
@@ -39,7 +39,7 @@ const importPrefix = "k8s.io/kubernetes/pkg/api"
 var accessor = meta.NewAccessor()
 
 // availableVersions lists all known external versions for this group from most preferred to least preferred
-var availableVersions = []unversioned.GroupVersion{v1.SchemeGroupVersion, v1beta3.SchemeGroupVersion}
+var availableVersions = []unversioned.GroupVersion{v1.SchemeGroupVersion}
 
 func init() {
 	registered.RegisterVersions(availableVersions)
-- 
1.9.3

