From 711edb06c665fa9a14e98ed7b0f1d943b26fd5ac Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Mon, 22 Feb 2016 10:11:46 +0100
Subject: [PATCH] hack hack/test-cmd.sh

---
 hack/lib/init.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hack/lib/init.sh b/hack/lib/init.sh
index 9fa47ca..1a18946 100644
--- a/hack/lib/init.sh
+++ b/hack/lib/init.sh
@@ -40,7 +40,7 @@ source "${KUBE_ROOT}/hack/lib/version.sh"
 source "${KUBE_ROOT}/hack/lib/golang.sh"
 source "${KUBE_ROOT}/hack/lib/etcd.sh"
 
-KUBE_OUTPUT_HOSTBIN="${KUBE_OUTPUT_BINPATH}/$(kube::util::host_platform)"
+KUBE_OUTPUT_HOSTBIN="/usr/bin"
 
 # emulates "readlink -f" which is not available on BSD (OS X).
 function readlinkdashf {
-- 
1.9.3

