From 0b94fc23c1c9ea2ad14fcbc9e436510d1992b2ca Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Mon, 29 Feb 2016 16:07:27 +0100
Subject: [PATCH] hyperkube-kubectl-dont shift os.Args

---
 cmd/hyperkube/kubectl.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmd/hyperkube/kubectl.go b/cmd/hyperkube/kubectl.go
index 9da15cf..954422d 100644
--- a/cmd/hyperkube/kubectl.go
+++ b/cmd/hyperkube/kubectl.go
@@ -28,7 +28,6 @@ func NewKubectlServer() *Server {
 		SimpleUsage: "Kubernetes command line client",
 		Long:        "Kubernetes command line client",
 		Run: func(s *Server, args []string) error {
-			os.Args = os.Args[1:]
 			if err := app.Run(); err != nil {
 				os.Exit(1)
 			}
-- 
1.9.3

