diff -up ppc64-diag-2.6.7/scripts/Makefile.script_loc ppc64-diag-2.6.7/scripts/Makefile
--- ppc64-diag-2.6.7/scripts/Makefile.script_loc	2014-08-25 11:10:34.368011105 +0200
+++ ppc64-diag-2.6.7/scripts/Makefile	2014-08-25 11:11:38.612012619 +0200
@@ -13,13 +13,13 @@ CFG_FILES = ppc64-diag.config
 all: $(SCRIPTS) $(INIT_FILES) $(CFG_FILES)
 
 install: all
-	@$(call install_files,$(SCRIPTS),744,$(DESTDIR)/etc/ppc64-diag)
+	@$(call install_files,$(SCRIPTS),744,$(DESTDIR)/usr/libexec/ppc64-diag)
 	@$(call install_files,$(INIT_FILES),755,$(DESTDIR)/$(LIBEXEC_DIR))
 	@$(call install_files,$(SERVICE_SCRIPT),644,$(DESTDIR)/$(SYSTEMD_DIR))
 	@$(call install_files,$(CFG_FILES),644,$(DESTDIR)/etc/ppc64-diag)
 
 uninstall: 
-	@$(call uninstall_files,$(SCRIPTS),$(DESTDIR)/etc/ppc64-diag)
+	@$(call uninstall_files,$(SCRIPTS),$(DESTDIR)/usr/libexec/ppc64-diag)
 	@$(call uninstall_files,$(INIT_FILES),$(DESTDIR)/$(LIBEXEC_DIR))
 	@$(call uninstall_files,$(SERVICE_SCRIPT),644,$(DESTDIR)/$(SYSTEMD_DIR))
 	@$(call uninstall_files,$(CFG_FILES),$(DESTDIR)/etc/ppc64-diag)
diff -up ppc64-diag-2.6.7/scripts/ppc64_diag_mkrsrc.script_loc ppc64-diag-2.6.7/scripts/ppc64_diag_mkrsrc
--- ppc64-diag-2.6.7/scripts/ppc64_diag_mkrsrc.script_loc	2014-08-16 07:35:55.000000000 +0200
+++ ppc64-diag-2.6.7/scripts/ppc64_diag_mkrsrc	2014-08-25 11:13:07.505014714 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/perl -I /etc/ppc64-diag
+#! /usr/bin/perl -I /usr/libexec/ppc64-diag
 #
 # This script is to be registered with servicelog as a notification tool.  It
 # is responsible for creating a new IBM.ServiceEvent RMC resource when a
@@ -14,7 +14,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
-require "/etc/ppc64-diag/servevent_parse.pl";
+require "/usr/libexec/ppc64-diag/servevent_parse.pl";
 
 # Check to make sure that devices.chrp.base.ServiceRM is installed
 $rc = system("which rpm >/dev/null 2>&1");
diff -up ppc64-diag-2.6.7/scripts/ppc64_diag_notify.script_loc ppc64-diag-2.6.7/scripts/ppc64_diag_notify
--- ppc64-diag-2.6.7/scripts/ppc64_diag_notify.script_loc	2014-08-16 07:35:55.000000000 +0200
+++ ppc64-diag-2.6.7/scripts/ppc64_diag_notify	2014-08-25 11:10:34.368011105 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/perl -I /etc/ppc64-diag
+#! /usr/bin/perl -I /usr/libexec/ppc64-diag
 #
 # This script is to be registered with servicelog as a notification tool.  It
 # is responsible for sending out e-mail notifications of new ppc64 platform
@@ -9,7 +9,7 @@
  
 use Getopt::Long;
 
-require "/etc/ppc64-diag/servevent_parse.pl";
+require "/usr/libexec/ppc64-diag/servevent_parse.pl";
 
 sub usage {
 	print "$0 [OPTIONS]\n";
diff -up ppc64-diag-2.6.7/scripts/ppc64_diag_servagent.script_loc ppc64-diag-2.6.7/scripts/ppc64_diag_servagent
--- ppc64-diag-2.6.7/scripts/ppc64_diag_servagent.script_loc	2014-08-16 07:35:55.000000000 +0200
+++ ppc64-diag-2.6.7/scripts/ppc64_diag_servagent	2014-08-25 11:10:34.369011105 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/perl -I /etc/ppc64-diag
+#! /usr/bin/perl -I /usr/libexec/ppc64-diag
 #
 # This script is to be registered with servicelog as a notification tool.  It
 # is responsible for invoking Electronic Service Agent in response to a
@@ -12,7 +12,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
  
-require "/etc/ppc64-diag/servevent_parse.pl";
+require "/usr/libexec/ppc64-diag/servevent_parse.pl";
 
 if (!-e "/usr/svcagent/bin/callsa2") {
 	# eSA is not installed; exit quietly
diff -up ppc64-diag-2.6.7/scripts/ppc64_diag_setup.script_loc ppc64-diag-2.6.7/scripts/ppc64_diag_setup
--- ppc64-diag-2.6.7/scripts/ppc64_diag_setup.script_loc	2014-08-16 07:35:55.000000000 +0200
+++ ppc64-diag-2.6.7/scripts/ppc64_diag_setup	2014-08-25 11:10:34.369011105 +0200
@@ -76,15 +76,15 @@ sub unregister {
 
 @notification_tools = (
 
-    ["/etc/ppc64-diag/ppc64_diag_notify -q -e root -l /var/log/platform",
+    ["/usr/libexec/ppc64-diag/ppc64_diag_notify -q -e root -l /var/log/platform",
      "--match='serviceable=1' ".
      "--type=EVENT --method=pairs_stdin"],
 
-    ["/etc/ppc64-diag/ppc64_diag_mkrsrc",
+    ["/usr/libexec/ppc64-diag/ppc64_diag_mkrsrc",
      "--match='serviceable=1' ".
      "--type=EVENT --method=pairs_stdin"],
 
-    ["/etc/ppc64-diag/ppc64_diag_migrate",
+    ["/usr/libexec/ppc64-diag/ppc64_diag_migrate",
      "--match=\'refcode=\"#MIGRATE\" and serviceable=0\' ".
      "--type=EVENT --method=pairs_stdin"],
 );
diff -up ppc64-diag-2.6.7/scripts/rtas_errd.script_loc ppc64-diag-2.6.7/scripts/rtas_errd
--- ppc64-diag-2.6.7/scripts/rtas_errd.script_loc	2014-08-25 11:10:34.369011105 +0200
+++ ppc64-diag-2.6.7/scripts/rtas_errd	2014-08-25 11:14:20.074016425 +0200
@@ -56,7 +56,7 @@ case "$1" in
 		echo "registering ppc64-diag with system"
 		rm -f /var/cache/ppc64-diag.registered
 		mkdir -v /var/cache/ppc64-diag.registered
-		/etc/ppc64-diag/ppc64_diag_setup --register
+		/usr/libexec/ppc64-diag/ppc64_diag_setup --register
 	fi
 	echo -n "Starting rtas_errd (platform error handling) daemon: "
 	if [ $INSSERV -eq 1 ]; then
