diff -up ppc64-diag-2.6.8/diags/Makefile.verbose ppc64-diag-2.6.8/diags/Makefile
--- ppc64-diag-2.6.8/diags/Makefile.verbose	2015-06-15 10:46:55.280425977 +0200
+++ ppc64-diag-2.6.8/diags/Makefile	2015-06-15 10:51:32.187388770 +0200
@@ -24,11 +24,11 @@ all: $(PROGS)
 
 diag_encl: $(DIAG_ENCL_OBJS)
 	@echo "LD $(WORK_DIR)/$@"
-	@$(CC) $(CFLAGS) $(LDFLAGS) $(INCLUDE) $(COMMON_UTILS_OBJ) -o $@ $^ $(DIAG_ENCL_LIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(INCLUDE) $(COMMON_UTILS_OBJ) -o $@ $^ $(DIAG_ENCL_LIBS)
 
 encl_led: $(ENCL_LED_OBJS)
 	@echo "LD $(WORK_DIR)/$@"
-	@$(CC) $(CFLAGS) $(LDFLAGS) $(INCLUDE) $(COMMON_UTILS_OBJ) -o $@ $^
+	$(CC) $(CFLAGS) $(LDFLAGS) $(INCLUDE) $(COMMON_UTILS_OBJ) -o $@ $^
 
 diag_encl.o: encl_util.h diag_encl.h
 
diff -up ppc64-diag-2.6.8/lpd/test/Makefile.verbose ppc64-diag-2.6.8/lpd/test/Makefile
--- ppc64-diag-2.6.8/lpd/test/Makefile.verbose	2015-05-11 05:18:51.000000000 +0200
+++ ppc64-diag-2.6.8/lpd/test/Makefile	2015-06-15 10:46:55.280425977 +0200
@@ -2,7 +2,6 @@
 # Makefile for lp/test
 #
 
-.SILENT:
 include ../../rules.mk
 
 CMDS = ledtool sesdevices
diff -up ppc64-diag-2.6.8/Makefile.verbose ppc64-diag-2.6.8/Makefile
--- ppc64-diag-2.6.8/Makefile.verbose	2015-06-15 10:46:55.280425977 +0200
+++ ppc64-diag-2.6.8/Makefile	2015-06-15 10:52:23.599563164 +0200
@@ -4,7 +4,6 @@
 
 include rules.mk
 
-.SILENT:
 BUILD_DIR=$(CURDIR)
 
 SUBDIRS = common rtas_errd diags scripts ela lpd opal_errd opal-dump-parse
diff -up ppc64-diag-2.6.8/rtas_errd/Makefile.verbose ppc64-diag-2.6.8/rtas_errd/Makefile
--- ppc64-diag-2.6.8/rtas_errd/Makefile.verbose	2015-05-11 05:18:51.000000000 +0200
+++ ppc64-diag-2.6.8/rtas_errd/Makefile	2015-06-15 10:46:55.280425977 +0200
@@ -30,11 +30,11 @@ rtas_errd: $(RTAS_ERRD_OBJS) $(COMMON_UT
 
 convert_dt_node_props: $(DT_NODE_OBJS)
 	@echo "LD $(WORK_DIR)/$@"
-	@$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^
 
 extract_platdump: $(EXTRACT_PLATDUMP_OBJS)
 	@echo "LD $(WORK_DIR)/$@"
-	@$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(EXTRACT_PLATDUMP_LIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(EXTRACT_PLATDUMP_LIBS)
 
 install: all
 	@$(call install_sbin,$(CMDS),$(DESTDIR))
diff -up ppc64-diag-2.6.8/rules.mk.verbose ppc64-diag-2.6.8/rules.mk
--- ppc64-diag-2.6.8/rules.mk.verbose	2015-05-11 05:18:51.000000000 +0200
+++ ppc64-diag-2.6.8/rules.mk	2015-06-15 10:46:55.280425977 +0200
@@ -211,7 +211,6 @@ CLEAN = @echo ""
 
 # Default target for building object files
 %.o: %.c
-	@echo "CC $(WORK_DIR)/$@"
-	@$(CC) -c $(AM_CFLAGS) $(CFLAGS) $(CPPFLAGS) $<
+	$(CC) -c $(AM_CFLAGS) $(CFLAGS) $(CPPFLAGS) $<
 
 
