diff -up ppc64-diag-2.6.9/diags/7031_D24_T24.c.unusevar ppc64-diag-2.6.9/diags/7031_D24_T24.c
--- ppc64-diag-2.6.9/diags/7031_D24_T24.c.unusevar	2015-06-29 12:37:42.000000000 +0200
+++ ppc64-diag-2.6.9/diags/7031_D24_T24.c	2015-07-01 09:38:45.540812529 +0200
@@ -73,7 +73,8 @@ get_enclosure_scsi_id(struct pearl_diag_
 static int
 pearl_print_drive_status(uint32_t status)
 {
-	int fail = 0, printed = 0, rc = OK;
+	__attribute__((unused)) int fail = 0;
+    int printed = 0, rc = OK;
 
 	if ((status & 0x0F000000) == 0x05000000) {
 		printf("(empty)  ");
@@ -171,7 +172,8 @@ print_ps_fan_status(uint32_t status)
 static int
 print_repeater_status(uint32_t status)
 {
-	int printed = 0, fail = 0, rc = OK;
+	int printed = 0, rc = OK;
+	__attribute__((unused)) int fail = 0;
 
 	if ((status & 0x0F000000) == 0x01000000)
 		printf("ok  ");
@@ -355,7 +357,8 @@ int
 diag_7031_D24_T24(int fd, struct dev_vpd *vpd)
 {
 	struct pearl_diag_page2 dp;
-	int failure = 0, rc, encl_id;
+	int failure = 0, encl_id;
+	__attribute__((unused)) int rc;
 	int buf_len = sizeof(dp);
 	int ps1, ps2, fan1, fan2, fan3, rpt, vpd_card;
 
diff -up ppc64-diag-2.6.9/rtas_errd/ela.c.unusevar ppc64-diag-2.6.9/rtas_errd/ela.c
--- ppc64-diag-2.6.9/rtas_errd/ela.c.unusevar	2015-06-29 12:37:42.000000000 +0200
+++ ppc64-diag-2.6.9/rtas_errd/ela.c	2015-07-01 09:16:26.435974668 +0200
@@ -1813,7 +1813,7 @@ process_v1_epow(struct event *event, int
 	int class;
 	int ext_epow;
 	int ext_epow_nofan;
-	int fan_num;
+	__attribute__((unused)) int fan_num;
 
 	/*
 	 * The following EPOW error logs found in CHRP, Ver. 1
diff -up ppc64-diag-2.6.9/rtas_errd/epow.c.unusevar ppc64-diag-2.6.9/rtas_errd/epow.c
--- ppc64-diag-2.6.9/rtas_errd/epow.c.unusevar	2015-06-29 12:37:42.000000000 +0200
+++ ppc64-diag-2.6.9/rtas_errd/epow.c	2015-07-01 09:16:26.436974672 +0200
@@ -48,7 +48,7 @@ static int time_remaining = 0;
 void
 epow_timer_handler(int sig, siginfo_t siginfo, void *context)
 {
-	int rc, state;
+    int __attribute__((unused)) rc, state;
 	struct itimerval tv;
 
 	if (time_remaining <= 0) {
@@ -149,7 +149,7 @@ parse_epow(struct event *event)
 	struct rtas_epow_scn *epow;
 	struct itimerval tv;
 	char	*event_type;
-	int	rc, state;
+	int __attribute__((unused)) rc, state;
 
 	/*
 	 * Check the sensor state;  this will be used to ensure
@@ -380,7 +380,7 @@ check_epow(struct event *event)
 {
 	pid_t	child;
 	char	*childargs[2];
-	int	rc, current_status;
+	int __attribute__((unused)) rc, current_status;
 
 	/*
 	 * Dissect the EPOW extended error information;
diff -up ppc64-diag-2.6.9/rtas_errd/files.c.unusevar ppc64-diag-2.6.9/rtas_errd/files.c
--- ppc64-diag-2.6.9/rtas_errd/files.c.unusevar	2015-06-29 12:37:42.000000000 +0200
+++ ppc64-diag-2.6.9/rtas_errd/files.c	2015-07-01 09:16:26.436974672 +0200
@@ -162,7 +162,7 @@ setup_rtas_event_scenario(void)
 {
 	struct stat	sbuf;
 	char	*tmp;
-	int	fd, len;
+	int	fd, __attribute__((unused)) len;
 	int	i;
 
 	if (scenario_file == NULL)
diff -up ppc64-diag-2.6.9/rtas_errd/guard.c.unusevar ppc64-diag-2.6.9/rtas_errd/guard.c
--- ppc64-diag-2.6.9/rtas_errd/guard.c.unusevar	2015-06-29 12:37:42.000000000 +0200
+++ ppc64-diag-2.6.9/rtas_errd/guard.c	2015-07-01 09:16:26.436974672 +0200
@@ -41,7 +41,7 @@ run_drmgr(enum resource_dealloc_type res
 	  unsigned int value, int wait)
 {
 	pid_t child;
-	int status, rc;
+	int status, __attribute__((unused)) rc;
 	char capacity[6], quant_str[5];
 	char *drmgr_args[] = {DRMGR_PROGRAM_NOPATH, "-r", "-c", NULL,
 			NULL, NULL, NULL, NULL, NULL};
