--- libservicelog/Makefile.am.libs	2016-10-17 09:25:44.930053462 -0400
+++ libservicelog/Makefile.am	2016-10-13 21:02:25.403271383 -0400
@@ -42,7 +42,7 @@ AM_YFLAGS	= -d -p v29nfy_
 AM_LFLAGS	= -Pv29nfy_ -olex.yy.c
 
 libservicelog_la_LDFLAGS = -module -version-info \
-	@GENERIC_API_VERSION@ -release @GENERIC_RELEASE@
+	@GENERIC_API_VERSION@ -release @GENERIC_RELEASE@ $(AM_LDFLAGS)
 
 LIBTOOL_DEPS = @LIBTOOL_DEPS@
 libtool: $(LIBTOOL_DEPS)
